starter-core/src/main/java/io/micronaut/starter/feature/database/Cassandra.java

```diff
@@ -53,6 +53,13 @@ public void apply(GeneratorContext generatorContext) {
                 .groupId("io.micronaut.cassandra")
                 .artifactId("micronaut-cassandra")
                 .compile());
+
+        if (generatorContext.isFeaturePresent(MicrometerFeature.class)) {
+            generatorContext.getConfiguration().put("cassandra.default.advanced.metrics.factory.class", "MicrometerMetricsFactory");
+            generatorContext.getConfiguration().put("cassandra.default.advanced.metrics.factory.class", "MicrometerMetricsFactory");
+            generatorContext.getConfiguration().put("cassandra.default.advanced.metrics.session.enabled", "[connected-nodes, cql-requests, bytes-sent, bytes-received]");
+            generatorContext.getConfiguration().put("cassandra.default.advanced.metrics.node.enabled", "[cql-requests, bytes-sent, bytes-received]");
```

Comment:
I don't think this will work when the config is in properties files.

I think the values need to be lists themselves, not string representation of lists.

Did you try creating an app using this?